build/orc-entrypoint.sh

```diff
@@ -16,18 +16,26 @@ fi
 sleep 10 # give time for SRV records to update
 
 NAMESPACE=$(</var/run/secrets/kubernetes.io/serviceaccount/namespace)
-jq -M ". + {
-        HTTPAdvertise:\"http://$HOSTNAME.$NAMESPACE:3000\",
-        RaftAdvertise:\"$HOSTNAME.$NAMESPACE\",
-        RaftBind:\"$HOSTNAME.$ORC_SERVICE.$NAMESPACE\",
-        RaftEnabled: ${RAFT_ENABLED:-"true"},
-        MySQLTopologyUseMutualTLS: true,
-        MySQLTopologySSLSkipVerify: true,
-        MySQLTopologySSLPrivateKeyFile:\"${ORC_CONF_PATH}/ssl/tls.key\",
-        MySQLTopologySSLCertFile:\"${ORC_CONF_PATH}/ssl/tls.crt\",
-        MySQLTopologySSLCAFile:\"${ORC_CONF_PATH}/ssl/ca.crt\",
-        RaftNodes:[]
-    }" "${ORC_CONF_FILE}" 1<>"${ORC_CONF_FILE}"
+jq -M \
+	--arg HTTPAdvertise "http://$HOSTNAME.$NAMESPACE:3000" \
+	--arg RaftAdvertise "$HOSTNAME.$NAMESPACE" \
+	--arg RaftBind "$HOSTNAME.$ORC_SERVICE.$NAMESPACE" \
+	--arg MySQLTopologySSLPrivateKeyFile "$ORC_CONF_PATH/ssl/tls.key" \
+	--arg MySQLTopologySSLCertFile "$ORC_CONF_PATH/ssl/tls.crt" \
+	--arg MySQLTopologySSLCAFile "$ORC_CONF_PATH/ssl/ca.crt" \
+	--argjson RaftEnabled "${RAFT_ENABLED:-true}" \
+	'. + {
+    HTTPAdvertise: $HTTPAdvertise,
+    RaftAdvertise: $RaftAdvertise,
+    RaftBind: $RaftBind,
+    RaftEnabled: $RaftEnabled,
+    MySQLTopologyUseMutualTLS: true,
+    MySQLTopologySSLSkipVerify: true,
+    MySQLTopologySSLPrivateKeyFile: $MySQLTopologySSLPrivateKeyFile,
+    MySQLTopologySSLCertFile: $MySQLTopologySSLCertFile,
+    MySQLTopologySSLCAFile: $MySQLTopologySSLCAFile,
+    RaftNodes: []
+  }' "$ORC_CONF_FILE" >"${ORC_CONF_FILE}.tmp" && mv "${ORC_CONF_FILE}.tmp" "$ORC_CONF_FILE"
```

Comment:
@nmarukovich can we use the same approach as in PXCO?
  like:
```
  local tmp=$(mktemp)
	sed "s/WRITE_NODE=.*/WRITE_NODE='$pod_zero.$service:3306'/g" /etc/proxysql-admin.cnf >${tmp}
	cat ${tmp} >/etc/proxysql-admin.cnf
```

Reply:
Probably yes. I can test it, but I like current way more, because it's easy to read and extend.